flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/data/DecimalDataUtils.java

```diff
@@ -208,23 +205,8 @@ public static boolean castToBoolean(DecimalData dec) {
         return dec.toBigDecimal().compareTo(BigDecimal.ZERO) != 0;
     }
 
-    public static TimestampData castToTimestamp(DecimalData dec) {
-        BigDecimal bd = dec.toBigDecimal().multiply(new BigDecimal(MILLS_PER_SECOND));
-        long mills = bd.longValue();
-        int nanos =
-                bd.remainder(BigDecimal.ONE)
-                        .multiply(new BigDecimal(NANOS_PER_MILLISECOND))
-                        .intValue();
-        return TimestampData.fromEpochMillis(mills, nanos);
-    }
-
-    public static DecimalData castFrom(TimestampData val, int precision, int scale) {
-        BigDecimal decimalSeconds =
-                new BigDecimal(val.getMillisecond())
-                        .multiply(new BigDecimal(NANOS_PER_MILLISECOND))
-                        .add(new BigDecimal(val.getNanoOfMillisecond()))
-                        .divide(new BigDecimal(NANOS_PER_SECOND));
-        return fromBigDecimal(decimalSeconds, precision, scale);
+    public static long castToTimestamp(DecimalData dec) {
```

Comment:
why change this?

Reply:
The original behavior before support bigint -> timestamp_ltz cast, 
 I think we can delete it and its tests, I checked it only used in tests after disable the cast, HDYT?

Reply:
We can simply delete it and add it back when necessary.